samples/HttpServer_FirmwareUpload/signtool.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python2
```

Comment:
The README.md must contain brief information how to create signed ROM and how to use it.

Reply:
The README.rst (already present in the folder) now contains the necessary instructions.